src/components/content-tab-sync.tsx

```diff
@@ -0,0 +1,187 @@
+			<div className="flex flex-col p-8">
+				<div className="flex items-center mb-1">
+					<div className="a8c-subtitle">{ __( 'Sync with' ) }</div>
+					<WordPressShortLogo className="ml-2 h-5" />
+				</div>
+				<div className="w-[40ch] text-a8c-gray-70 a8c-body pr-2">
+					{ __(
+						'Connect an existing WordPress.com site, or create a new one and share your site with the world.'
+					) }
+				</div>
+				<div className="mt-6">
+					{ [
+						__( 'Push and pull changes from your live site.' ),
+						__( 'Supports staging and production sites.' ),
+						__( 'Sync database and file changes.' ),
+					].map( ( text ) => (
+						<div key={ text } className="text-a8c-gray-70 a8c-body flex items-center">
+							<Icon className="fill-a8c-blueberry mr-2 shrink-0" icon={ check } /> { text }
+						</div>
+					) ) }
+				</div>
+				{ children }
+			</div>
+			<div className="flex flex-col shrink-0 items-end p-4">
+				<SyncTabImage />
+			</div>
+		</div>
+	);
+}
+
+function NoSyncedSites( { selectedSite }: React.ComponentProps< typeof SiteSyncDesription > ) {
+	return (
+		<SiteSyncDesription selectedSite={ selectedSite }>
+			<CreateConnectSite className="mt-8" selectedSite={ selectedSite } />
+		</SiteSyncDesription>
+	);
+}
+
+// TO DO: Implement use selectedSite prop once we have a way to determine if the user has any sites connected
+// Remove the eslint-disable-next-line comment once the prop is used
+
+function CreateConnectSite( {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	selectedSite,
+	className = '',
+}: {
+	selectedSite: SiteDetails;
+	className?: string;
+	tagline?: string;
+} ) {
+	const { __ } = useI18n();
+	const isOffline = useOffline();
+
+	const offlineMessageCreate = __( 'Creating new site requires an internet connection.' );
+	const offlineMessageConnect = __( 'Connecting a site requires an internet connection.' );
+
+	return (
+		<div className={ className }>
+			<div className="flex gap-4">
+				<Tooltip disabled={ ! isOffline } text={ offlineMessageCreate }>
+					<Button
+						aria-disabled={ isOffline }
+						variant="primary"
+						onClick={ () => {
+							if ( isOffline ) {
+								return;
+							}
+
+							// TO DO: Open the site creation flow.
+						} }
+					>
+						{ __( 'Create new site' ) }
+						<span className="ltr:ml-1 rtl:mr-1 rtl:scale-x-[-1]">↗</span>
+					</Button>
+				</Tooltip>
+				<Tooltip disabled={ ! isOffline } text={ offlineMessageConnect }>
+					<Button
+						aria-disabled={ isOffline }
+						variant="secondary"
+						className={ cx( ! isOffline && '!text-a8c-blueberry !shadow-a8c-blueberry' ) }
+						onClick={ () => {
+							if ( isOffline ) {
+								return;
+							}
+
+							// TO DO: Open the modal to connect a site.
+						} }
+					>
+						{ __( 'Connect site' ) }
+					</Button>
+				</Tooltip>
+			</div>
+		</div>
+	);
+}
+
+function NoAuthSyncTab( { selectedSite }: React.ComponentProps< typeof SiteSyncDesription > ) {
```

Comment:
Note: there is some repetition here with the Share tab logic for when the user is not logged in. I have thought about extracting it into a reusable component but it seems that it is our plan to remove `Share` tab in the future so I was not sure it was worth it. 

Happy to do it if it is a better approach.